src/net.h

```diff
@@ -79,15 +79,25 @@ static const bool DEFAULT_BLOCKSONLY = false;
 static const int64_t DEFAULT_PEER_CONNECT_TIMEOUT = 60;
 /** Number of file descriptors required for message capture **/
 static const int NUM_FDS_MESSAGE_CAPTURE = 1;
+/** The default setting for dandelion transactions */
+static const bool DEFAULT_DANDELION = true;
```

Comment:
One last thing before I ACK this PR:

**We should consider having Dandelion disabled by default.**
Otherwise we will lose DigiByte's instant transactions because all transactions will have to enter the `stempool` first. People can still enable the protocol by passing `-dandelion 1` when launching their own nodes.
The iOS wallet supports this too by broadcasting protocol internal `dandelion_tx` transactions.

@JaredTate @gto90 @barrystyle @ycagel 
Would love to hear your thoughts about this.

By the way, this will fix the CI tests too.
Alternatively we can add `gArgs.ForceSetArg("-dandelion", "0");` to the `wallet_tests`

Reply:
Forget what I just said. It's fine when it's enabled by default.

@barrystyle Can you add another commit in `wallet_tests.cpp` at line 700?

`gArgs.ForceSetArg("-dandelion", "0");`

Reply:
I think it is beneficial to leave Dandelion enabled by default, to grant new users extra privacy by default. Perhaps it could be added as a popup during the install process? Giving the user direct control. Not sure what all that would entail as it would take GUI work too. From my experience even with dandelion txs are still quick enough to seem fast.